src/shards-info.cr

```diff
@@ -17,6 +19,15 @@ Kemal::Session.config do |config|
   config.secret = "my_super_secret"
 end
 
+Raven.configure do |config|
+  config.async = true
+  config.current_environment = Kemal.config.env
+  config.dsn = ENV["SENTRY_DNS"]
```

Comment:
You shouldn't need to set `config.dsn` manually if you set `SENTRY_DSN` env variable.